### PR TITLE
Symfony 5 Compatibility

### PR DESCRIPTION
**Pull request by mathieupetrini**
I am targeting this branch because master because I had to upgrade sonata-admin-bundle to is dev-master version (until release v4.0)

## Changelog

```markdown
### Added
- Symfony 5 Compatibility

### Removed
- Behat Tests
```

## Subject

Symfony 5 Compatibility

### CODE REVIEW

**Comment by mathieupetrini** [comment]
@kunicmarko20 clean PR from master to master.

Sorry for the inconvenience.

**Comment by mathieupetrini** [comment]
@lukepass (https://github.com/kunicmarko20/SonataAnnotationBundle/pull/39#issuecomment-721581962)

PHP 7.2 is still preserved.

But I think we should wait for Sonata 4.0 version which will be the next version Symfony 5 compatible.
I'm not sure Sonata 3 will be compatible.

**Comment by mathieupetrini** [comment]
> @lukepass ([#39 (comment)](https://github.com/kunicmarko20/SonataAnnotationBundle/pull/39#issuecomment-721581962))
> 
> PHP 7.2 is still preserved.
> 
> But I think we should wait for Sonata 4.0 version which will be the next version Symfony 5 compatible.
> I'm not sure Sonata 3 will be compatible.

https://scrutinizer-ci.com/g/kunicmarko20/SonataAnnotationBundle/inspections/2a41c9a0-f0c6-4758-a3d3-8798fff3ed37 ok i understood PHP 7.2. We cannot keep compatiblity in favor of DoctrineOrmAdminBundle:dev-master ...

File: composer.json [hunk 1]
```diff
@@ -18,32 +18,24 @@
...
     "doctrine/annotations": "^1.6",
-    "sonata-project/admin-bundle": "^3.35",
-    "symfony/config": "^3.4 || ^4.0",
-    "symfony/dependency-injection": "^3.4 || ^4.0",
-    "symfony/finder": "^3.4 || ^4.0",
-    "symfony/http-kernel": "^3.4 || ^4.0"
-  },
-  "conflict": {
-    "sonata-project/admin-bundle": "<3.35"
+    "sonata-project/admin-bundle": "dev-master",
```
Comment (kunicmarko20) [inline composer.json:23]: why is it like? don't think we can release it like this, because this is unstable

File: composer.json [hunk 2]
```diff
@@ -18,32 +18,24 @@
...
+    "symfony/config": "^4.0 || ^5.0",
```
Comment (kunicmarko20) [inline composer.json]: should be `4.4 || 5.1`

**Comment by kunicmarko20** [comment]
@mathieupetrini Thank you for this PR, really nice work! I left a few comments, and I have a question, why did you remove Behat tests?

File: composer.json (hunk 2 above)
Comment (mathieupetrini) [reply composer.json]: yes, you're right. fixed.

File: composer.json (hunk 1 above)
Comment (mathieupetrini) [reply composer.json:23]: Yes, we cannot release until sonata release theirs v4 versions which will be the version Symfony 5 compatible.

**Comment by mathieupetrini** [comment]
> @mathieupetrini Thank you for this PR, really nice work! I left a few comments, and I have a question, why did you remove Behat tests?

I remove Behat because there are many packages seems not maintained : 

- https://github.com/Behat/MinkExtension/blob/v2.2/composer.json
- https://github.com/minkphp/MinkBrowserKitDriver/blob/v1.3.4/composer.json
- https://github.com/minkphp/MinkGoutteDriver/blob/v1.2.1/composer.json

I think it is difficult to keep the compatibility.

**Comment by kunicmarko20** [comment]
Hello @mathieupetrini, thank you for this PR but unfortunately, I decided I am gonna archive all of sonata related repositories since I do not have time to maintain them, and I didn't use SonataAdmin in over 3 years.

**Pull request closed without merging**